modules/custom/social_media_system/social_media_system.install

```diff
@@ -20,3 +20,17 @@ function social_media_system_update_12101(): void {
     $config_storage->write('core.entity_view_mode.media.featured', $data);
   }
 }
+
+/**
+ * Change uri_scheme of field_media_file from public to private.
+ */
+function social_media_system_update_12102(): string {
+  /** @var \Drupal\update_helper\Updater $updater */
+  $updater = \Drupal::service('update_helper.updater');
+
+  // Execute configuration update definitions with logging of success.
+  $updater->executeUpdate('social_media_system', 'social_media_system_update_121002');
```

Comment:
typo - replace `social_media_system_update_121002` with `social_media_system_update_12102` or with `__FUNCTION__`

Reply:
Thanks @nechai, I've fixed a typo.
You should be able to test it locally and you can ignore failed CI, since it's not related.